### PR TITLE
Enable nullability in MenuCommands

### PR DESCRIPTION
**Pull request by halgab**
This one is pretty trivial

## Proposed changes

- Enable nullability in `MenuCommands`

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/dotnet/winforms/pull/10072)

### CODE REVIEW

File: src/System.Windows.Forms.Design/src/PublicAPI.Shipped.txt [hunk 1]
```diff
@@ -85,41 +85,41 @@ override System.Windows.Forms.Design.MaskDescriptor.ToString() -> string!
...
-~static readonly System.Windows.Forms.Design.MenuCommands.KeySizeHeightDecrease -> System.ComponentModel.Design.CommandID
-~static readonly System.Windows.Forms.Design.MenuCommands.KeySizeHeightIncrease -> System.ComponentModel.Design.CommandID
-~static readonly System.Windows.Forms.Design.MenuCommands.KeySizeWidthDecrease -> System.ComponentModel.Design.CommandID
-~static readonly System.Windows.Forms.Design.MenuCommands.KeySizeWidthIncrease -> System.ComponentModel.Design.CommandID
-~static readonly System.Windows.Forms.Design.MenuCommands.KeyTabOrderSelect -> System.ComponentModel.Design.CommandID
-~static readonly System.Windows.Forms.Design.MenuCommands.SelectionMenu -> System.ComponentModel.Design.CommandID
-~static readonly System.Windows.Forms.Design.MenuCommands.SetStatusRectangle -> System.ComponentModel.Design.CommandID
-~static readonly System.Windows.Forms.Design.MenuCommands.SetStatusText -> System.ComponentModel.Design.CommandID
-~static readonly System.Windows.Forms.Design.MenuCommands.TraySelectionMenu -> System.ComponentModel.Design.CommandID
+static readonly System.Windows.Forms.Design.MenuCommands.ComponentTrayMenu -> System.ComponentModel.Design.CommandID!
```
Comment (dreddy-work) [inline src/System.Windows.Forms.Design/src/PublicAPI.Shipped.txt:88]: why these changes? just because we enabled nullability?

Comment (halgab) [reply src/System.Windows.Forms.Design/src/PublicAPI.Shipped.txt:88]: Exactly. Nothing is nullable, so the file itself barely changes

**Review by dreddy-work — APPROVED** [review]
